assets/src/features/room-page/components/PinComponents.tsx

```diff
@@ -12,22 +12,24 @@ export const PinTileLayer: FC<PinUserButtonProps> = ({ pinned, onClick }: PinUse
   const pinText = pinned ? "Unpin" : "Pin";
 
   return (
-    <Button
-      className={clsx(
-        "absolute top-1/2 left-1/2 -translate-x-1/2 -translate-y-1/2",
-        "hidden group-hover:flex",
-        "flex-row content-center",
-        "opacity-50 hover:opacity-75",
-        "gap-2 py-4 px-8",
-        "rounded-full bg-brand-grey-100"
-      )}
-      onClick={onClick}
-    >
-      <Pin className="text-brand-white" />
-      <span className={"font-lg font-aktivGrotesk font-semibold leading-6 tracking-wide text-brand-white"}>
-        {pinText}
-      </span>
-    </Button>
+    <div className="group absolute h-full w-full">
```

Comment:
I had to remove the "group" class from the tile because hovering over a tile now displays Pinning button. Simulcast options also react to hover because of tooltips. So I moved this group to avoid collisions.

Tailwind 3.0 introduces a nested groups, but we still have tailwind 2
https://tailwindcss.com/docs/hover-focus-and-other-states#differentiating-nested-groups

Reply:
OK. Nice to mention: there is [a ticket](https://membraneframework.atlassian.net/browse/MV-319) about this update.